src/misc/lv_mem.c

```diff
@@ -204,6 +205,36 @@ void * lv_mem_realloc(void * data_p, size_t new_size)
     return new_p;
 }
 
+lv_res_t lv_mem_realloc_safe(void ** data, size_t new_size)
+{
+    void * new_p = NULL;
+
+    if(data == NULL) {
+        LV_LOG_ERROR("Invalid input");
+        return LV_RES_INV;
+    }
+
+    MEM_TRACE("Safely reallocating %p with %lu size", *data, (unsigned long) new_size);
+
+#if LV_MEM_CUSTOM == 0
+    new_p = lv_tlsf_realloc(tlsf, data, new_size);
```

Comment:
You need existing address, so it must be `*data` in parameter

Reply:
Done in 503e757